tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -14,18 +14,18 @@
 class Homescreen(Base):
 
     name = 'Default Home Screen'
-    manifest_url = '{}verticalhome{}/manifest.webapp'.format(Base.DEFAULT_PROTOCOL,Base.DEFAULT_APP_HOSTNAME)
 
     _homescreen_icon_locator = (By.CSS_SELECTOR, 'gaia-grid .icon')
     _homescreen_all_icons_locator = (By.CSS_SELECTOR, 'gaia-grid .icon:not(.placeholder)')
     _homescreen_collection_icon = (By.CSS_SELECTOR, 'gaia-grid .collection')
     _edit_mode_locator = (By.CSS_SELECTOR, 'body.edit-mode')
     _search_bar_icon_locator = (By.ID, 'search-input')
-    _landing_page_locator = (By.ID, 'icons')
+    _landing_page_locator = (By.ID, 'apps')
```

Comment:
you're right, i should remove it, i didn't delete it after using it for the method that didn't work out.